documentation/src/main/wiki/SourceCollector.md

```diff
@@ -6,7 +6,7 @@ Module name: **source**
 
 | ! Note    |
 | :-------- |
-| System is waiting up to 20 seconds before request is timed out. This parameter is configurable via OSGi configuration (`AET Source Collector Factory`). |
+| System is waiting up to 40 seconds before request is timed out. This parameter is configurable via OSGi configuration (`AET Source Collector Factory`). |
```

Comment:
Are you sure that it is switched to 40 seconds?
It looks like you forgot to update [SourceCollectorFactory](https://github.com/Cognifide/aet/blob/master/core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/source/SourceCollectorFactory.java) `DEFAULT_TIMEOUT` constant.